src/clientStub.ts

```diff
@@ -59,10 +69,8 @@ export class DgraphClientStub {
             return Promise.reject("Invalid op argument in alter");
         }
 
-        return this.callAPI("alter", {
-            method: "POST",
-            body,
-        });
+        const config = {method: "POST", body, ...this.options};
```

Comment:
I'd inline the `config` variable as 
`this.callAPI("alter", {
  foo: 1,
  bar: 'baz',
})`
and I would probably put `...this.options` as the first entry, to make sure options cannot overwrite method or body. 
Though I cannot think of a scenario where `options` with `method` or `body` would be a good idea.

Reply:
fixed